LEAF_Request_Portal/sources/Workflow.php

```diff
@@ -567,32 +568,36 @@ public function updateStep($stepID, $stepTitle, $bgColor = '', $fontColor = '')
         return 1;
     }
 
-    public function saveStepData($stepID, $data){
+    public function saveStepData(int $stepID, array $data) : bool{
+
+        $validSaveStep = TRUE;
 
         // everything that seems to modify this stuff is run through here.
         if (!$this->login->checkGroup(1))
         {
-            return 'Admin access required.';
```

Comment:
PSR12 requires the { to be on the same line as the conditional statement. Unless, the conditions are on separate lines in which case the ) { need to be on there own line following the conditions.

Reply:
Updated this and any other entries I could find.